core/src/main/java/com/linecorp/armeria/client/HttpClientFactory.java

```diff
@@ -166,6 +167,7 @@ final class HttpClientFactory implements ClientFactory {
         this.options = options;
 
         clientDelegate = new HttpClientDelegate(this, addressResolverGroup);
+        RequestTargetCache.registerClientMetrics(meterRegistry);
```

Comment:
I split the cache into server- and client-side ones because they parse request targets differently (absolute forms and fragments notably).